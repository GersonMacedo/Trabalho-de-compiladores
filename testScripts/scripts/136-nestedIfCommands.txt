! -s SYNTACTIC
! should accept nested if else commands
program p ;
begin
    if a then
        if b then
            if c then
                t := 3
            else
                t := 2
        else
            if c then
                t := 2
            else
                t := 1
    else
        if b then
            if c then
                t := 2
            else
                t := 1
        else
            if c then
                t := 1
            else
                t := 0
    ;
end .