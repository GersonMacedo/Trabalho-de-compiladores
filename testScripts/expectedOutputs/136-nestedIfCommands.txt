
Performing up to syntactic analysis at testScripts/scripts/136-nestedIfCommands.txt
No errors found
